docs/content/maintainers/first-issue-template.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "First Issue Template"
+date: 2018-08-25T20:00:24-07:00
+draft: false
+---
+
+This document provides a template for any issue body that is to be labeled
+`good first issue`. This should assume the contributor is either new to open
+source or at least new to the project.
```

Comment:
@robjloranger would it be ok to state here that this document is for issue authors? I'm concerned that folks might read this doc and misunderstand what it's for

Reply:
Yes absolutely, I will elaborate more here. As well it will likely be the maintainer assigned or taking care of issue triage.